dosbox-staging/dosbox-staging
# Remove obsolete Intel iGPU workaround

# Description

https://github.com/dosbox-staging/dosbox-staging/blob/c04107d084c1f4f51919a31ee1b68f59fde0d437/src/gui/sdlmain.cpp#L1348-L1361
This workaround for Intel iGPUs was added a few years back after I reported an issue where Dosbox OpenGL output was dark on 10th-gen Intel iGPU. Recently, I have re-tested OpenGL renderer without this workaround on 11th-gen Intel with the latest graphics driver and the issue was not reproduced. This means that this workaround is possibly obsolete but we need someone with 6th-gen to 10th-gen Intel to confirm this.


## Related issues

https://github.com/dosbox-staging/dosbox-staging/pull/1829


# Manual testing

Tested on 11th-gen Intel iGPU with 32.0.101.6078 graphics driver:
https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html

Need someone with 6th-gen to 10th-gen Intel iGPU to test OpenGL renderer in PR build and see if it appears correctly. Make sure you have the latest graphics driver:
https://www.intel.com/content/www/us/en/download/776137/intel-7th-10th-gen-processor-graphics-windows.html


# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:
